### PR TITLE
fixed eslint settings - not enabling typescript linting yet

### PR DESCRIPTION
**Pull request by scottysseus**
- fixed the linter settings and linted the frontend code
- we can add typescript support for the linter if you guys want

### CODE REVIEW

File: package.json
Comment (hypevhs) [inline package.json]: npm WARN deprecated babel-eslint@10.1.0: babel-eslint is now @babel/eslint-parser. This package will no longer receive updates.

Thoughts?

**Comment by hypevhs** [comment]
Yesterday you were enabling the linter in VSCode. Maybe add instructions on how to do that, to the README. Unless the process has changed as a result of this PR and you don't know.

File: webpack.config.common.js
Comment (hypevhs) [inline webpack.config.common.js]: These imports are unused. Can we remove them since .babelrc does all the preset stuff?

File: webpack.config.common.js
Comment (hypevhs) [inline webpack.config.common.js]:
```suggestion
          exclude: /node_modules/,
```

File: webpack.config.common.js
Comment (hypevhs) [inline webpack.config.common.js]: It seems `'*'` is no longer necessary to resolve imports with explicit extensions.
```suggestion
      extensions: ['.js', '.jsx', '.tsx', '.ts']
```

File: package.json
Comment (scottysseus) [reply package.json]: I changes this to @babel/eslint-parser

**Comment by scottysseus** [comment]
@libjared I added linting instructions to the README

**Review by hypevhs — APPROVED** [review]
LGTM after a few nitpicks. After this merges I want to PR some eslint/webpack/babel stuff.

**Pull request merged**
